skfem/models/general.py

```diff
@@ -15,3 +15,9 @@ def divergence(u, v, w):
 def rot(v, w):
     return np.einsum('i...,ij...,j...',
                      w.w, np.array([[0, 1], [-1, 0]]), grad(v))
+
+
+@LinearForm
+def vrot(v, w):
+    return np.einsum('i...,ij...,j...',
```

Comment:
Using the extended definition of `curl` in #507, can this be written as `dot(curl(v), w.w)`?

Reply:
Yes, that's much nicer!  Thanks!